parent/pom.xml

```diff
@@ -247,7 +247,8 @@
         <cxf.oauth.bundle.version>20100527_1</cxf.oauth.bundle.version>
         <cxf.oro.bundle.version>2.0.8_6</cxf.oro.bundle.version>
         <cxf.osgi.saaj.version>[1.4,2)</cxf.osgi.saaj.version>
-        <cxf.osgi.version>5.0.0</cxf.osgi.version>
+        <cxf.osgi.version>6.0.0</cxf.osgi.version>
+        <cxf.osgi.compendium.version>5.0.0</cxf.osgi.compendium.version>
```

Comment:
Why not upgrading to 6.0.0 as well using the updated coordinates ?

Reply:
changed to artifact osgi.cmpn